File: Accounting101.Angular/Accounting101.Angular.Server/Models/LoginModel.cs
namespace Accounting101.Angular.Server.Models;

public class LoginModel
{
    public string Email { get; set; } = string.Empty;

    public string Password { get; set; } = string.Empty;

    public string MultiFactorAuthenticationCode { get; set; } = string.Empty;

    public string MultiFactorAuthenticationResetCode { get; set; } = string.Empty;
}

File: Accounting101.Angular/accounting101.angular.client/src/app/models/login.model.ts
export class LoginModel {
  public email: string;
  public password: string;
  public twoFactorAuthenticationCode: string;
  public twoFactorAuthenticationCodeReset: string;

  constructor() {
    this.email = '';
    this.password = '';
    this.twoFactorAuthenticationCode = '';
    this.twoFactorAuthenticationCodeReset = '';
  }
}
